rp

```diff
@@ -44,9 +44,14 @@ dbg() {
 }
 
 
-add_warning_tag() {
+detect_git_dir() {
```

Comment:
I think you've based your commit on the wrong branch. This change looks a bit out of the place here.

Reply:
Nope; I intentionally added both commits. Creating two PRs is a bunch of extra work and it does no harm :shrug: